docs/components/Input.md

```diff
@@ -79,6 +79,10 @@ export default {
         </template>
       </template>
     </VInput>
+
+    <button type="submit" :disabled="!form.valid">
```

Comment:
My guess is that the `disabled` attribute was copied from `VForm` examples, but it doesnt actually work here.